include/zephyr/drivers/spi/spi_nor.h

```diff
@@ -0,0 +1,134 @@
+#define SPI_NOR_CMD_PP_4B        0x12  /* Page Program 4 Byte Address */
+#define SPI_NOR_CMD_PP_1_1_4_4B  0x34  /* Quad Page program (1-1-4) 4 Byte Address */
+#define SPI_NOR_CMD_PP_1_4_4_4B  0x3e  /* Quad Page program (1-4-4) 4 Byte Address */
+#define SPI_NOR_CMD_RDFLSR       0x70  /* Read Flag Status Register */
+#define SPI_NOR_CMD_CLRFLSR      0x50  /* Clear Flag Status Register */
+#define SPI_NOR_CMD_WR_VCFGREG   0x81  /* Octal Write volatile configuration Register */
+#define SPI_NOR_OCMD_READ        0xFD  /* Octal IO read command */
+
+/* Flash octal opcodes */
+#define SPI_NOR_OCMD_SE         0x21DE  /* Octal Sector erase */
+#define SPI_NOR_OCMD_CE         0xC738  /* Octal Chip erase */
+#define SPI_NOR_OCMD_RDSR       0x05FA  /* Octal Read status register */
+#define SPI_NOR_OCMD_DTR_RD     0xEE11  /* Octal IO DTR read command */
+#define SPI_NOR_OCMD_RD         0xEC13  /* Octal IO read command */
+#define SPI_NOR_OCMD_PAGE_PRG   0x12ED  /* Octal Page Prog */
+#define SPI_NOR_OCMD_WREN       0x06F9  /* Octal Write enable */
+#define SPI_NOR_OCMD_NOP        0x00FF  /* Octal No operation */
+#define SPI_NOR_OCMD_RESET_EN   0x6699  /* Octal Reset Enable */
+#define SPI_NOR_OCMD_RESET_MEM  0x9966  /* Octal Reset Memory */
+#define SPI_NOR_OCMD_WR_CFGREG2 0x728D  /* Octal Write configuration Register2 */
+#define SPI_NOR_OCMD_RD_CFGREG2 0x718E  /* Octal Read configuration Register2 */
+#define SPI_NOR_OCMD_BULKE      0x609F  /* Octa Bulk Erase */
+
+ /* Page, sector, and block size are standard, not configurable. */
+#define SPI_NOR_PAGE_SIZE      0x0100U
+#define SPI_NOR_SECTOR_SIZE    0x1000U
+#define SPI_NOR_BLOCK_32K_SIZE 0x8000U
+#define SPI_NOR_BLOCK_SIZE     0x10000U
+
+/* Flash Auto-polling values */
+#define SPI_NOR_WREN_MATCH    0x02
+#define SPI_NOR_WREN_MASK     0x02
+
+#define SPI_NOR_WEL_MATCH     0x00
+#define SPI_NOR_WEL_MASK      0x02
+
+#define SPI_NOR_MEM_RDY_MATCH 0x00
+#define SPI_NOR_MEM_RDY_MASK  0x01
+
+#define SPI_NOR_AUTO_POLLING_INTERVAL   0x10
+
+/* Flash Dummy Cycles values */
+#define SPI_NOR_DUMMY_RD                8U
+#define SPI_NOR_DUMMY_RD_OCTAL          6U
+#define SPI_NOR_DUMMY_RD_OCTAL_DTR      6U
+#define SPI_NOR_DUMMY_REG_OCTAL         4U
+#define SPI_NOR_DUMMY_REG_OCTAL_DTR     5U
+
+
+/* Memory registers address */
+#define SPI_NOR_REG2_ADDR1              0x0000000
+#define SPI_NOR_CR2_STR_OPI_EN          0x01
+#define SPI_NOR_CR2_DTR_OPI_EN          0x02
+#define SPI_NOR_REG2_ADDR3              0x00000300
+#define SPI_NOR_CR2_DUMMY_CYCLES_66MHZ  0x07
+
+/* Test whether offset is aligned to a given number of bits. */
+#define SPI_NOR_IS_ALIGNED(_ofs, _bits) (((_ofs) & BIT_MASK(_bits)) == 0)
+#define SPI_NOR_IS_SECTOR_ALIGNED(_ofs) SPI_NOR_IS_ALIGNED(_ofs, 12)
+#define SPI_NOR_IS_32K_ALIGNED(_ofs) SPI_NOR_IS_ALIGNED(_ofs, 15)
+#define SPI_NOR_IS_64K_ALIGNED(_ofs) SPI_NOR_IS_ALIGNED(_ofs, 16)
+
+#define CMD_RDCR 0x15 /* Read the configuration register. */
```

Comment:
For consistency, this macro should be renamed `SPI_NOT_CMD_RDCR`.

For that purpose, I think you can squash commits "include: spi: Move spi_nor.h to include folder"
and "drivers: flash: Adjust include paths for spi_nor.h".